includes/class-wc-payments.php

```diff
@@ -1257,6 +1257,15 @@ public static function get_customer_service_api(): WC_Payments_Customer_Service_
 		return self::$customer_service_api;
 	}
 
+	/**
+	 * Returns the order service instance.
+	 *
+	 * @return WC_Payments_Order_Service
+	 */
+	public static function get_order_service(): WC_Payments_Order_Service {
+		return self::$order_service;
```

Comment:
I noticed in PR description that we still use the legacy Order Service. I think that mixing from the 'old' way and the 'new' way could harm us eventually. Is there a chance to migrate this into its own dedicated service without relaying on legacy one?

Reply:
I'm glad you caught this, as it is one of the topics I'd like to discuss. Now, the approach here is objective. There are a few scenarios:

1. "Just" migrate the whole service to `src`. However, if we want to stick to order IDs instead of objects, as you've noticed we're doing in this series of PRs, we'd need to change quite a bit of code, and that would be quite the effort, which we don't have time for presently.
2. Use the new service for calls with `$order_id` and new code, leaving the migration to legacy code for some point in the future. I'm ok with this approach.
3. Copy code from the existing service into the new one. This will lead to code duplication, which until we get to migrate the whole service, might become unmanageable.

Reply:
It kind of is, but also not: I am using the new `OrderService` as a wrapper for the old one, but it's not just that: New functionality is added to the new service. That's option 2.